l/llvmlite/build_info.json

```diff
@@ -1,18 +1,16 @@
 {
-   "maintainer": "npanpaliya",
-   "package_name": "llvmlite",
-   "github_url": "https://github.com/numba/llvmlite",
-   "version": "*",
-   "default_branch": "main",
-   "build_script": "llvmlite_v0.43.0_ubi_9_3.sh",
-   "package_dir": "l/llvmlite",
-   "docker_build": false,
-   "validate_build_script": true,
-   "use_non_root_user": false,
-   "v0.41.*": {
-       "build_script": "llvmlite_v0.41.1_ubi_9_3.sh"
-   },
-   "v0.43.*": {
-       "build_script": "llvmlite_v0.43.0_ubi_9_3.sh"
-   }
-}
+  "maintainer": "aastha-sharma2",
+  "package_name": "llvmlite",
+  "github_url": "https://github.com/numba/llvmlite",
+  "version": "v0.40.0",
+  "default_branch": "main",
+  "wheel_build" : true,
+  "package_dir": "l/llvmlite",
+  "build_script": "llvmlite_ubi_9.3.sh",
+  "docker_build": false,
+  "validate_build_script": true,
+  "use_non_root_user": false,
+  "*": {
```

Comment:
@aastha-sharma2 we should not remove support for already supported version, Keep existing mapping of version and build script file intact.